opensquirrel/ir.py

```diff
@@ -189,21 +206,22 @@ class Statement(IRNode, ABC):
 
 
 class Measure(Statement, ABC):
-
     def __init__(
         self,
+        bit: Bit,
```

Comment:
I think we should follow the Qiskit convention of the order of the parameters, _i.e._, first the qubit that is measure and then the bit into which the measurement outcome is stored: see https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.QuantumCircuit#measure

Reply:
Done!